.editorconfig

```diff
@@ -4,6 +4,7 @@ root = true
 ; Unix-style newlines
```

Comment:
Just a minor suggestion: We could extend the comment , this is from the official docs: `Unix-style newlines with a newline ending every file`

Reply:
I think we ca do without, the settings speaks for itself. Also no-one ever looks at that file ;)